server/bin/pbench-base.sh

```diff
@@ -138,3 +138,23 @@ function quarantine () {
         fi
     done
 }
+
+function log_info {
+    echo $1
+}
+
+function log_error {
```

Comment:
Can we consider the following for `log_error` instead, so that we don't need `status_append_and_log`?

```
function log_error {
    if [[ -z "${2}" ]]; then
        printf -- "%s\n" "${1}" >&4
    else
        printf -- "%s\n" "${1}" | tee -a "${2}" >&4
    fi
}
```

It is more mechanism to use `printf` instead of `echo`, but it avoids the implied behavior of `echo` with new lines.

This also lets us use `log_error` to specify a file to also add the line to, like:

```
log_error "my error message" "${mail_content}"
```

Where "`my error message`" is printed to `>&4` and to the file name in `${mail_content}`.

Reply:
`log_info` will have to get the same treatment.

Reply:
Not sure why github won't let me reply to the two comments that ask whether we log the two messages today: the answer is we do log them today; they are sent to stdout within the log_init/log_finish section, so they are redirected to the log file.

Reply:
Are there any places we have informational logs saved to the report content?

Reply:
Not in `pench-unpack-tarballs` - maybe there are some in the other scripts, but we can leave `log_info` simple, until and unless we need to complicate it.